infra/modules/bastion/main.tf

```diff
@@ -68,7 +68,7 @@ resource "aws_instance" "bastion" {
   instance_type               = "t3.small" # Anything lower and npm install can crash the instance
   subnet_id                   = var.public_subnet
   vpc_security_group_ids      = [aws_security_group.ec2_sg.id]
-  associate_public_ip_address = true
+  associate_public_ip_address = false
```

Comment:
don't need the IP?

Reply:
We just connect to them through the AWS console in an SSM session now, so I made it false. This was actually to match the way it exists as deployed, the setting was out of date somehow

Reply:
Ah lovely, you should be able to do it using AWS CLI too using SSM sessions